### PR TITLE
bitcoin: update to 0.20.1

### PR DESCRIPTION
**Pull request by ndowens**

### CODE REVIEW

File: srcpkgs/bitcoin/template [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 pkgname=bitcoin
-version=0.19.1
-revision=3
+version=0.20.1
+revision=1
 build_style=gnu-configure
 configure_args="--with-incompatible-bdb --disable-ccache --disable-static
  --enable-hardening --with-boost=${XBPS_CROSS_BASE}/usr"
-hostmakedepends="pkg-config yasm protobuf"
+hostmakedepends="pkg-config yasm protobuf autoconf automake libtool"
```
Comment (ofiala-a51) [inline srcpkgs/bitcoin/template:8]: those are already implicitly required by setting the `build_style` to `gnu-configure`

Comment (ndowens) [reply srcpkgs/bitcoin/template:8]: It's already set as gnu-configure and it doesn't pull them

Comment (ericonr) [reply srcpkgs/bitcoin/template:8]: Those are additional, yeah. Autohell welcomes you :)

**Comment by ev-ermakov** [comment]
There are no need for `protobuf`, `protobuf-devel` and `libprotoc-devel` anymore.

**Comment by ev-ermakov** [comment]
Also, seems the patch `qt5.15.patch` can be removed.

**Comment by ev-ermakov** [comment]
The following code block should be added:
```sh
pre_build() {
	export BITCOIN_GENBUILD_NO_GIT=1
}
```

And just a little cosmetic change:
```diff
-		vinstall contrib/${b}.bash-completion 644 usr/share/bash-completion $b
+		vcompletion contrib/${b}.bash-completion bash $b
```

**Pull request closed without merging**